appendix/backup-and-restore/docker-compose.rst

```diff
@@ -0,0 +1,151 @@
+Backup & Restore (Docker)
+=========================
+
+This section shows some basics about the backup and restore process for a docker
+compose based deployment of Zammad. If you already use a different method or
+tool, it is fine to stick with it as long as it works. In such a case, consider
+using the :ref:`disable-backup-service scenario <additional-scenarios>`.
+
+If you're familiar with docker, the Quick Start section below includes the
```

Comment:
We should say that the rest of the document describes the built- in backup and restore mechanism.